pyproject.toml

```diff
@@ -7,10 +7,11 @@ dependencies = [
     "craft-cli>=2.6.0",
     "craft-grammar@git+https://github.com/canonical/craft-grammar@feature/pydantic-2",
     "craft-parts@git+https://github.com/canonical/craft-parts@feature/2.0",
-    "craft-providers@git+https://github.com/canonical/craft-providers@feature/2.0-git-modules",
+    "craft-providers@git+https://github.com/canonical/craft-providers@feature/2.0",
```

Comment:
I'm not sure which one is better (this one has more recent changes and `green` pipelines)

Reply:
The `git-modules` one was just because charmcraft was having failures trying to install that one due to a git submodule issue. We should be using the one you have :-)